csharp/test/Ice/optional/Test.ice

```diff
@@ -39,7 +39,7 @@ struct VarStruct
 
 sequence<byte> ByteSeq;
 sequence<bool> BoolSeq;
-sequence<short> ShortSeq;
+["cs:generic:List"] sequence<short> ShortSeq;
```

Comment:
No cs:generic for sequence of built-in tests was a gap in our testings that revealed some bugs.